lib/trento/event_handlers_supervisor.ex

```diff
@@ -5,6 +5,8 @@ defmodule Trento.EventHandlersSupervisor do
 
   alias Trento.Infrastructure.Commanded.EventHandlers.{
     AlertsEventHandler,
+    DatabaseDeregistrationEventHandler,
```

Comment:
Personally, I would put both events in the same handler, as they are tightly related.
But it is personal choice. Implementation wise both are correct

Reply:
We can, maybe I will add a tech debt for that, in my mind was a better idea, but yeah the events are pretty related`